### PR TITLE
Add primes algorithm for go

### PR DESCRIPTION
**Pull request by Stratus51**

### CODE REVIEW

File: primes/go/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ARG VERSION
+ARG DISTRIBUTION
+FROM golang:${VERSION}${DISTRIBUTION}
```
Comment (bbc2) [inline primes/go/Dockerfile]:
```suggestion
FROM golang:${VERSION}-${DISTRIBUTION}
```

File: primes/go/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /app
+
+COPY . /app
+
+RUN go build
+
+ENTRYPOINT ./go
```
Comment (bbc2) [inline primes/go/Dockerfile]:
```suggestion
ENTRYPOINT ./app
```

**Review by bbc2 — CHANGES_REQUESTED** [review]

File: primes/go/Dockerfile (hunk 1 above)
Comment (Stratus51) [reply primes/go/Dockerfile]: Fixed

File: primes/go/Dockerfile (hunk 2 above)
Comment (Stratus51) [reply primes/go/Dockerfile]: Fixed with proper build executable name specification (-o build option)

**Review by bbc2 — APPROVED** [review]

**Pull request merged**